PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -2331,6 +2331,7 @@ function Get-SddcDiagnosticInfo
                 # _C_ token will be replaced with node fqdn for cimsession/computername callouts
                 # _N_ token will be replaced with node non-fqdn
                 $CmdsToLog =
+                            @{ C = 'Get-ClusterResourceType'; F = $null },

```

Comment:
Note where these commands are run - one job per cluster node resulting in a .XML/TXT drop to the per node gather directories, running on the node invoking P.D (which doesn't need to be a cluster node - important!). This is why they require remoting.

This one should move up to where the other Get-Cluster* whole-cluster gathers are (not per node!), using $AccessNode. That block of gathers could very well be condensed into a form similar to this

Reply:
Yeah, let me double check the requirements what they need for get-clusterresourcetype.  Feels like we can run it on a per cluster instance